frontend/src/pages/CourseSelector/CourseSelector.tsx

```diff
@@ -23,6 +24,7 @@ const CourseSelector = () => {
 
   const dispatch = useDispatch();
 
+  const courseResCache = useRef({} as CourseDescInfoResCache);
```

Comment:
nit can use more specific var name here too, but if u cbbs it's ok

Reply:
k